docs/operator/configuration.md

```diff
@@ -16,6 +16,7 @@ Configuration of the operator's Pod is done via environment variables at startup
 | `OPERATOR_CIS_KUBERNETES_BENCHMARK_ENABLED` | `true`                 | The flag to enable CIS Kubernetes Benchmark scanner |
 | `OPERATOR_VULNERABILITY_SCANNER_ENABLED`    | `true`                 | The flag to enable vulnerability scanner |
 | `OPERATOR_CONFIG_AUDIT_SCANNER_ENABLED`     | `true`                 | The flag to enable configuration audit scanner |
+| `OPERATOR_VULNERABILITY_SCANNER_LATEST`     | `false`                | The flag to enable vulnerability scanner to only check the latest replicaset |
```

Comment:
The `_LATEST` is not very meaningful here without going to the docs. What if we call it `OPERATOR_VULNERABILITY_SCANNER_SCAN_ALL_REPLICASETS` or `OPERATOR_VULNERABILITY_SCANNER_SCAN_ONLY_CURRENT_REVISIONS` or (your idea goes here ... )

Reply:
Used OPERATOR_VULNERABILITY_SCANNER_SCAN_ONLY_CURRENT_REVISIONS, using OPERATOR_VULNERABILITY_SCANNER_SCAN_ALL_REPLICASETS and setting that one to false would feel so strange and I can see how people would missinturpet it.